html/elements/input/week.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-week": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/week",
+            "description": "<code>type=\"week\"</code>",
+            "support": {
+              "chrome": {
+                "version_added": "20"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a>."
```

Comment:
One last nit, sorry. Can we change this note to only "See <a href='https://bugzil.la/888320'>bug 888320</a>." Thank you, @maboa!